gum/arch-arm64/gumarm64writer.c

```diff
@@ -220,6 +220,7 @@ gum_arm64_writer_init (GumArm64Writer * writer,
   writer->label_defs = NULL;
   writer->label_refs.data = NULL;
   writer->literal_refs.data = NULL;
+  writer->data_endian = GUM_ENDIAN_NATIVE;
```

Comment:
Assignment should go just before `target_os`.

Reply:
Done.